### DISCUSSION TITLE
John 7.38

### DISCUSSION THREAD

**Discussion by AZBuckeye** [openenglishbible/Open-English-Bible#346] [Q&A] [answered]
I notice that OEB's translation of John 7.38 appears to be significantly different from other translations.  Why is that?

**Comment by openenglishbible** [comment, marked as answer]
There is a very simple answer to that - it is an error! Thanks for pointing it out! What appears to have happened is John 8:38 has been transposed on top of John 7:38, which should read:

> If any one is thirsty, let him come to me and drink. From the heart of those who believe in me will flow, as is said in scripture, rivers of living water.

I have fixed in the sources, and will generate a new version with the fix included soon.

Thanks again!

**Comment by AZBuckeye** [comment]
Well, thanks. I made a mistake once, and, who knows, that may happen again some time!

Sent from Yahoo Mail on Android 
 
  On Sun, Jun 26, 2022 at 6:34 PM, Open English ***@***.***> wrote:   


There is a very simple answer to that - it is an error! Thanks for pointing it out! What appears to have happened is John 8:38 has been transposed on top of John 7:38, which should read:


If any one is thirsty, let him come to me and drink. From the heart of those who believe in me will flow, as is said in scripture, rivers of living water.


I have fixed in the sources, and will generate a new version with the fix included soon.

Thanks again!
